src/utils/quick-open.js

```diff
@@ -96,6 +104,31 @@ export function formatSymbols(
   };
 }
 
+export function formatGeneralQuery(): Array<FormattedGeneralQuery> {
+  const queryResult = [
+    {
+      value: "Seach for a function in a file",
+      title: "@ Function Search",
```

Comment:
we should use l10n here
https://github.com/devtools-html/debugger.html/blob/master/docs/local-development.md#l10n

Reply:
Seems like this is also the source of those `id.123` values too.